scripts/sanitize.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
```

Comment:
its intended to be both python2 and 3 since it may get run on trusty, though I'll admit to not testing as much on Trusty.